test/repositories/slick/implementations/ChatsRepositorySpec.scala

```diff
@@ -1040,9 +1106,9 @@ class ChatsRepositorySpec extends AsyncWordSpec with OptionValues with MustMatch
           optChatOverBCC <- chatsRep.getChat(basicTestDB.chatRow.chatId, bccOverseerUserRow.userId)
 
         } yield assert(
-          optChatOverFrom === visibleBCCOptChat &&
-            optChatOverTo === notVisibleBCCOptChat &&
-            optChatOverCC === notVisibleBCCOptChat &&
+          optChatOverFrom === visibleBCCOptChat &
```

Comment:
Why did you replace "&&" with "&" everywhere? I believe they are not the same thing. The same way we use "||" (and not "|") for the logical operator "or".
https://www.tutorialspoint.com/scala/scala_operators.htm

Reply:
Because && fails immediately, this means that after one comparison fails, you lose information about which other comparisons failed which makes debugging harder.